### ISSUE TITLE
Forgs mearching

### ISSUE THREAD

**Issue by Olum-hack** [Amazon-Vine-Explorer/AmazonVineExplorer#45]
First thank you for developing that tool
I can see that some other also try to improve the tool.
for example:
- [https://github.com/deburau/AmazonVineExplorer](https://github.com/deburau/AmazonVineExplorer)
- [https://github.com/richardsevern/AmazonVineExplorer-UK](https://github.com/richardsevern/AmazonVineExplorer-UK)
- [https://github.com/Olum-hack/AmazonVineExplorer](https://github.com/Olum-hack/AmazonVineExplorer)

It would be create to merge them again and improve the main.
What do you think?

**Comment by deburau** [comment]
Good point! But until the original author comments on this, I will continue working on my own fork. And by the way, I merged your changes in my fork. Seeing the status of the background scanner is a really nice feature.

**Comment by MarkusSR1984** [comment]
Hi, sorry for my late reaction, as i wrote in the readme, i´m not able to continue the project. But i will handle it over to someone who want to do this. 
deburau i will add you to the org that u can do changes on the original repository, thx for your work
